src/plugins.mjs

```diff
@@ -0,0 +1,33 @@
+export const plugins = [];

```

Comment:
would it be cleaner to abstract this plugins array with a registerPlugins
so we could do all the registration check (typeof == function etc...) at registration time

(just a suggestion)

Reply:
I thought about that as well, but then we'd also need methods to unregister a plugin and change the execution order in case two plugins modify the same entry point data. I think everyone should be familiar with modifying an array, so we should be fine leaving this without abstraction.